test/parallel/test-http.js

```diff
@@ -10,7 +10,7 @@ var body0 = '';
 var body1 = '';
 
 var server = http.Server(function(req, res) {
-  if (responses_sent == 0) {
+  if (responses_sent === 0) {
     assert.equal('GET', req.method);
```

Comment:
Can you update `assert.equal()` here and elsewhere to use `assert.strictEqual()` instead? Might as well get all our minor refactoring in at once. :-D